pkg/ui/workspaces/db-console/src/app.spec.tsx

```diff
@@ -25,678 +25,706 @@ stubComponentInModule("src/views/statements/statementsPage", "default");
-import { ClusterVisualization } from "src/views/clusterviz/containers/map";
-import { NodeGraphs } from "src/views/cluster/containers/nodeGraphs";
-import { NodeOverview } from "src/views/cluster/containers/nodeOverview";
-import { Logs } from "src/views/cluster/containers/nodeLogs";
-import { EventPageUnconnected } from "src/views/cluster/containers/events";
-import { DatabasesPage } from "src/views/databases/databasesPage";
-import { DatabaseDetailsPage } from "src/views/databases/databaseDetailsPage";
-import { DatabaseTablePage } from "src/views/databases/databaseTablePage";
-import DataDistributionPageConnected from "src/views/cluster/containers/dataDistribution";
-import SQLActivityPage from "src/views/sqlActivity/sqlActivityPage";
-import StatementsPage from "src/views/statements/statementsPage";
-import TransactionsPage from "src/views/transactions/transactionsPage";
-import {
-  JobsPage,
-  StatementDetails,
-  TransactionDetails,
-  ActiveStatementDetails,
-  ActiveTransactionDetails,
-} from "@cockroachlabs/cluster-ui";
-import Debug from "src/views/reports/containers/debug";
-import { ReduxDebug } from "src/views/reports/containers/redux";
-import { CustomChart } from "src/views/reports/containers/customChart";
-import { EnqueueRange } from "src/views/reports/containers/enqueueRange";
-import { RangesMain } from "src/views/devtools/containers/raftRanges";
-import { RaftMessages } from "src/views/devtools/containers/raftMessages";
-import Raft from "src/views/devtools/containers/raft";
-import NotFound from "src/views/app/components/errorMessage/notFound";
-import { ProblemRanges } from "src/views/reports/containers/problemRanges";
-import { Localities } from "src/views/reports/containers/localities";
-import { Nodes } from "src/views/reports/containers/nodes";
-import { DecommissionedNodeHistory } from "src/views/reports";
-import { Network } from "src/views/reports/containers/network";
-import { Settings } from "src/views/reports/containers/settings";
-import { Certificates } from "src/views/reports/containers/certificates";
-import { Range } from "src/views/reports/containers/range";
-import { Stores } from "src/views/reports/containers/stores";
-
 describe("Routing to", () => {
   let history: MemoryHistory<unknown>;
-  let appWrapper: ReturnType<typeof mount>;
 
   beforeEach(() => {
     history = createMemoryHistory({
       initialEntries: ["/"],
     });
     const store: Store<AdminUIState, Action> = createAdminUIStore(history);
-    appWrapper = mount(<App history={history} store={store} />);
-  });
-
-  afterEach(() => {
-    appWrapper.unmount();
+    render(<App history={history} store={store} />);
   });
 
   const navigateToPath = (path: string) => {
     history.push(path);
-    appWrapper.update();
   };
 
   describe("'/' path", () => {
-    it("routes to <ClusterOverview> component", () => {
+    test("routes to <ClusterOverview> component", () => {
```

Comment:
Any reason to change these from `it` to `test`?  One is aliased to the other in Jest (though I can't remember which one), so they should be equivalent.